build.gradle

```diff
@@ -13,8 +13,8 @@ allprojects {
 subprojects {
     apply plugin: 'java'
 
-    sourceCompatibility = "1.8"
-    targetCompatibility = "1.8"
+    sourceCompatibility = "1.11"
+    targetCompatibility = "1.11"
```

Comment:
We're going to have to continue compiling for 1.8 to support java 8 for the short term.